server/src/main/java/org/opensearch/search/profile/query/InternalProfileCollector.java

```diff
@@ -68,13 +68,46 @@ public class InternalProfileCollector implements Collector, InternalProfileCompo
     /** The wrapped collector */
     private final ProfileCollector collector;
 
+    /** The elapsed time in the concurrent search reduce phase */
+    private final Long reduceTime;
+
+    /** The maximum slice time for this collector manager */
+    private final Long maxSliceTime;
+
+    /** The minimum slice time for this collector manager */
+    private final Long minSliceTime;
+
+    /** The average slice time for this collector manager */
+    private final Long avgSliceTime;
+
+    /** The segment slice count for this collector manager */
+    private final int sliceCount;
```

Comment:
Not needed in this class

Reply:
Thank you for reviewing the changes. I have now addressed them accordingly.